packages/jest-jasmine2/src/index.js

```diff
@@ -283,6 +284,18 @@ function jasmine2(
     }
   });
 
+  const jasmineExpect = env.expect;
+
+  // extend (and replace) the returned value from `expect` with `jest-matchers`
+  // values. This can be a performance bottleneck, but right now it just takes
+  // 0.015s per `expect` call
```

Comment:
just remove this. We'll make this better eventually and we are doing this a lot for Jasmine.